backend/pkg/handlers/stripe_webhook.go

```diff
@@ -0,0 +1,129 @@
+package handlers
+
+import (
+	"Learning-Mode-AI/pkg/config"
+	"Learning-Mode-AI/pkg/services"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/stripe/stripe-go/customer"
+	"github.com/stripe/stripe-go/v81"
+	"github.com/stripe/stripe-go/v81/webhook"
+)
+
+const MaxBodyBytes = int64(65536)
+
+func StripeWebhookHandler(endpointSecret string) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+
+		var productTierMap = map[string]string{
+			config.ProductIdPro: "Pro", // Our product id in stripe account test mode
```

Comment:
can you remove this comment?
```suggestion
			config.ProductIdPro: "Pro", 
```